Assets/Scripts/IInteractable.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public interface IInteractable
+{
+    void Interact(GameObject interacter);
+    void LightIt(GameObject interacter);
```

Comment:
LightIt is specific method and should no be in generic IInteractable, why it is bad?
What if you will have obcject that can't be lighted? problems...
I would recommend create another Interface ILightable and move this method there. Scripts can inherit by many interfaces